# move_base.yaml
base_global_planner: "navfn/NavfnROS"
base_local_planner: "dwa_local_planner/DWAPlannerROS"

# Global Costmap
global_costmap:
  global_frame: "map"
  robot_base_frame: "base_link"
  update_frequency: 5.0
  static_map: true

# Local Costmap
local_costmap:
  global_frame: "odom"
  robot_base_frame: "base_link"
  update_frequency: 5.0
  rolling_window: true

DWAPlannerROS:
  max_vel_x: 0.5
  min_vel_x: 0.1
  max_rot_vel: 1.0
  acc_lim_x: 1.0
  acc_lim_theta: 2.0

